joi/package.json

```diff
@@ -0,0 +1,17 @@
+{
+  "name": "joi",
+  "amdName": "hookformResolversJoi",
+  "version": "1.0.0",
+  "private": true,
+  "description": "React Hook Form validation resolver: joi",
+  "main": "dist/joi.js",
+  "module": "dist/joi.module.js",
+  "umd:main": "dist/joi.umd.js",
+  "source": "src/index.ts",
+  "types": "dist/index.d.ts",
+  "license": "MIT",
+  "peerDependencies": {
+    "react-hook-form": ">=6.6.0",
+    "@hookform/resolvers": "^1.0.0"
```

Comment:
dependent itself?

Reply:
Yes, to import `convertArrayToPathName`, inspiration from [preact](https://github.com/preactjs/preact/blob/master/hooks/package.json#L21)

Reply:
oh cool! 👍